include/dma.h

```diff
@@ -130,7 +130,7 @@ struct dma_block_config {
  *
  * dma_callback is the callback function pointer. If enabled, callback function
  *              will be invoked at transfer completion or when error happens
- *              (error_code: zero-transfer success, non zero-error happens).
+ *              (ret_code: non negative-transfer success, nenative-error happens).
  */
```

Comment:
pls fix this typo: 'nenative' into 'negative'.

Reply:
I've rewritten the comments again to minimize the affect of the changes: define non-zero return value as the one defined by a specific driver since some DMA drivers give positive values as errors as well.  As for STM32 DMA, let's define negative as errors, and positive as how many bytes RX DMA has received.